pkgs/by-name/rd/rdp/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rdp";
+  version = "0.13.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "urschrei";
+    repo = "rdp";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DqkifIzctC/f6as/X/+njb2aCFmf4yB1Nj/LMh2IQSY=";
+  };
+
+  cargoLock.lockFile = ./Cargo.lock;
```

Comment:
See if you can refactor this into `useFetchCargoVendor` before we start adding more Cargo.locks to the tree.

Reply:
unfortunately now cargo.lock on the repo